bundle/Resources/views/admin/dashboard/index.html.twig

```diff
@@ -1,43 +1,50 @@
 {% extends "@NovaeZMailing/admin/layout/pagelayout.html.twig" %}
 
+{% trans_default_domain 'ezmailing' %}
+
+
 {% block novezmailing_breadcrumb %}
     <ol class="breadcrumb px-4">
         <li class="breadcrumb-item active">
             Nova eZ Mailing
         </li>
     </ol>
 {% endblock %}
-{% block novezmailing_title %}Dashboard{% endblock %}
-{% block novezmailing_subtitle %}Quick overview{% endblock %}
+
+
+{% set title = "dashboard.title"|trans %}
```

Comment:
you can put it back normally. This is an eZ bug: https://jira.ez.no/browse/EZP-29902